protocol/src/common/protocol.ts

```diff
@@ -2807,7 +2811,9 @@ export {
 	// Semantic Tokene
 	SemanticTokenTypes, SemanticTokenModifiers, SemanticTokensLegend, SemanticTokens, SemanticTokensPartialResult, SemanticTokensEdit, SemanticTokensDelta,
 	SemanticTokensDeltaPartialResult, TokenFormat, SemanticTokensClientCapabilities, SemanticTokensOptions, SemanticTokensRegistrationOptions, SemanticTokensParams,
-	SemanticTokensRequest, SemanticTokensDeltaParams, SemanticTokensDeltaRequest, SemanticTokensRangeParams, SemanticTokensRangeRequest, SemanticTokensRefreshNotification
+	SemanticTokensRequest, SemanticTokensDeltaParams, SemanticTokensDeltaRequest, SemanticTokensRangeParams, SemanticTokensRangeRequest, SemanticTokensRefreshNotification,
+	// Resolve moniker support
+	Moniker, ResolveMonikerParams, ResolveMonikerRegistrationOptions, ResolveMonikerRequest
```

Comment:
You can't export this here since it is a proposed feature. This needs to go into the api.ts file first in the proposed namespace. You can look into this version https://github.com/microsoft/vscode-languageserver-node/blob/66f91b3583a19ae621214d908ce19c9b2d182626/protocol/src/common/api.ts